src/amo/components/LanguagePicker.js

```diff
@@ -1,14 +1,59 @@
+/* global window */
+import config from 'config';
 import React, { PropTypes } from 'react';
 import { compose } from 'redux';
+import { connect } from 'react-redux';
 
+import { languageJSON } from 'core/languages';
```

Comment:
I doubt we're going to add to that file so this could probably be `import languages from 'core/languages';`.